<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>738</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>578</width>
    <height>738</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>578</width>
    <height>738</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Начать тест</string>
  </property>
  <widget class="QLabel" name="topicLabel">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>50</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string> </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timer">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="timerLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Продолжительность теста</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tasks">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tasksLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Количество заданий</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>461</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Начать тест</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
